scss/components/button.scss

```diff
@@ -60,6 +60,7 @@ $block: ns(button);
         padding-right: tn-space(4);
     }
     &--large {
+        @include tn-type(2, header, semi);
```

Comment:
You don't have to specify the font face or weight if there is no change from the base. This can be `tn-type(2)` only which will increase the font size and not output redundant rules.